wasm/index.html

```diff
@@ -9,33 +9,64 @@
         background: -webkit-linear-gradient(to top, #A1FFCE, #FAFFD1);
         background: linear-gradient(to top, #A1FFCE, #FAFFD1);
       }
-      h1 {
-        text-align: center;
-        font-family: "Monaco", "Andale Mono", "Lucida Console", "FreeMono", monospace;
-      }
-      p {
+      body, h1, p {
         text-align: center;
         font-family: "Monaco", "Andale Mono", "Lucida Console", "FreeMono", monospace;
       }
       .badge {
-        color: #fff;
-        background-color: #000;
+        color: #eee;
+        background-color: #111;
         display: inline-block;
         padding: .25em .3em;
         line-height: 1;
         border-radius: .25rem;
         font-weight: 700;
         text-decoration: none;
       }
-      #gotherworld {
+      .badge-invert {
+        color: #111;
+        background-color: #eee;
```

Comment:
we could use the badge-invert to show which part and debug level is selected as well.